metadata.rb

```diff
@@ -6,5 +6,5 @@
 long_description 'Installs/Configures deploy-user'
 version '1.4.2'
 
-depends 'ssh_known_hosts', '~> 0.2'
+depends 'ssh_known_hosts', '~> 4.0'
```

Comment:
https://github.com/inviqa/chef-deploy-user/blob/master/Berksfile#L9-L11 this being what is causing the custom cookbook to be fetched rather than official one